src/drivers/express/ExpressAdminRouteDriver.ts

```diff
@@ -203,5 +215,17 @@ export default class ExpressAdminRouteDriver {
           },
         }),
       );
+
+
+  // Lambda routes
+    router.post(
+      '/change-author',
```

Comment:
We should change this to be /users/:username/learning-objects/:cuid/change-author to be more restful.

Reply:
The username being the current username of the author of the learning object.